docs/concepts/protocol/06-airstreams.mdx

```diff
@@ -70,23 +71,37 @@ in to prove that they are eligible, and create a stream if they are. That's all
 Additionally, you don't have to immediately fund the Airstream contract. You can just create the contract and at a later
 date fund it with the airdropped tokens.
 
-**This has three great implications:**
+**This has four great implications:**
 
 - **Recipients pay for the gas fees themselves to create the stream**, when they claim (the claim action creates the
   stream). Creating a campaign with thousands of recipients would be incredibly costly if you had to pay for all the gas
   fees.
+- **You don't have to fund the campaign from the get go**. You can deploy the campaign contract now, and fund it at a
+  later date when you feel comfortable doing so. There is no rush.
 - **You keep full control over unclaimed Airstreams**. If a recipient doesn't claim their Airstream, it's not created,
   and you remain in full control over their allocation.
   - This applies only if the campaign has an expiration date. If there is no expiration date, the recipients can claim
     their Airstreams at any time in the future.
-- **You don't have to fund the campaign from the get go**. You can deploy the campaign contract now, and fund it at a
-  later date when you feel comfortable doing so. There is no rush.
+- **You can retrieve your funds in case of misconfiguration**. There is a grace period during which you can retrieve
+  unclaimed funds. The grace period ends 7 days after the first claim is made. This is very useful in case where you
+  have incorrectly configured the Airstream campaign or deposited more tokens than required.
 
 Once the Airstream campaign is launched, recipients who have claimed can withdraw the tokens which have already been
 streamed at any time using our interface at [app.sablier.com](https://app.sablier.com).
 
+## Types of Airstreams
+
+We offer two types of Airstream campaigns:
```

Comment:
In general, we should prefer non-personal formulations such as `There are` over `We offer`

cc @smol-ninja @andreivladbrg

Reply:
I agree. "There are two types of Airstream campaigns:"?? If you are making changes already, feel free to change it.

Reply:
https://github.com/sablier-labs/docs/pull/138/commits/eed3c918e5e832f4ed13a3380f96f1d8a7bf9dda